qa/Rakefile

```diff
@@ -0,0 +1,66 @@
+require 'rspec'
+require 'rspec/core/rake_task'
+
+task :spec    => 'spec:all'
+task :default => :spec
+
+namespace :test do
+
+  def fetch_config
+    require "open3"
+    Open3.popen3("vagrant ssh-config") do |stdin, stdout, stderr, wait_thr|
+      { :stdout => stdout.read.chomp, :stderr => stderr.read.chomp,
+        :exit_status => wait_thr.value.exitstatus }
+    end
+  end
```

Comment:
Maybe add a return a message if vagrant is not installed or not available in the current $PATH?

by curiosity, since vagrant is written in ruby, do they have a gem that wrap vagrant? (Not really needed, was a simple curiosity)